test/instrumentation/modules/aws-sdk/dynamodb.test.js

```diff
@@ -1,3 +1,5 @@
+'use strict'
```

Comment:
+1 here -- it'd be nice if our linter/CI caught these though (not for this PR, just something to keep in mind for the future)

Reply:
Yah, agreed. I *think* we can further configure our `standard` usage with eslint rules?  https://eslint.org/docs/rules/strict